website/content/vagrant-cloud/hcp-vagrant/post-migration-guide.mdx

```diff
@@ -70,3 +70,9 @@ If all organizations have been migrated to HCP Vagrant Registry, only the HCP ac
 ```
 export VAGRANT_CLOUD_TOKEN=";$(hcp auth print-access-token)"
 ```
+
+# Packer
+
+After migrating to HCP box uploads may fail when using the [vagrant-cloud](/packer/integrations/hashicorp/vagrant/latest/components/post-processor/vagrant-cloud) post-processor. This is due to the HCP access token used in the composite token expiring prior to the post-processor being executed.
```

Comment:
Feel free to disregard/I'm not sure this is right, but this felt slightly confusing on first read without a comma.

```suggestion
After migrating to HCP, box uploads may fail when using the [vagrant-cloud](/packer/integrations/hashicorp/vagrant/latest/components/post-processor/vagrant-cloud) post-processor. This is due to the HCP access token used in the composite token expiring prior to the post-processor being executed.
```